src/App.tsx

```diff
@@ -1,61 +1,29 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Form } from './components/Form/Form';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { getUserById } from './services/user';
+
+export const initialTodos = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
```

Comment:
The logic for calculating the new `id` for todos should be moved to the `App` component. Currently, the `id` is calculated in the `Form` component, which contradicts the requirement to handle data preparation in the `App` component.